internal/cloud/metadata.go

```diff
@@ -113,6 +127,13 @@ func (i *Identifier) IdentifyCloudProvider() (string, error) {
 		return GCP, nil
 	}
 
+	if result, err := i.identifyNutanix(); err != nil {
+		return "", err
+	} else if result {
+		log.Infof("VM is running on %s", Nutanix)
+		return Nutanix, nil
+	}
+
 	log.Info("VM is not running in any recognized cloud provider")
```

Comment:
Same here

Reply:
As above
Referring now to a `system` 752cf74eb7e2d23bc9e5b6c04e18503c3a3bffc9